docs/source/python_api.rst

```diff
@@ -22,21 +22,21 @@ Using internal mamba APIs
 
 The core of ``mamba`` is written in C++, but we expose the internals of mamba with a Python API (using ``pybind11``).
 
-You can import the ``mamba_api`` using ``from mamba import mamba_api``.
+You can import ``libmambapy`` containing the Python API using ``from mamba.api import libmambapy``.
```

Comment:
I think `import libmambapy` works as well (not necessary to have `mamba` installed).